startup/20-area-detectors.py

```diff
@@ -350,20 +357,28 @@ def stage(self):
 #pilatus300 section is unmarked.  032018-MF
 #pilatus300 section is marked out for bluesky upgrade.  010819-RL
 time.sleep(1)
-pilatus300 = PilatusV33('XF:11BMB-ES{Det:SAXS}:', name='pilatus300')
-pilatus300.tiff.read_attrs = []
-pilatus300.stats3.total.kind = 'hinted'
-pilatus300.stats4.total.kind = 'hinted'
-STATS_NAMES = ['stats1', 'stats2', 'stats3', 'stats4', 'stats5']
-pilatus300.read_attrs = ['tiff'] + STATS_NAMES
-for stats_name in STATS_NAMES:
-    stats_plugin = getattr(pilatus300, stats_name)
-    stats_plugin.read_attrs = ['total']
-#pilatus300.cam.ensure_nonblocking()
-
-#pilatus800 section
 
+#if True:
+#if True:
 if False:
+    pilatus300 = Pilatus300V33('XF:11BMB-ES{Det:SAXS}:', name='pilatus300')
+    #pilatus300 = PilatusV33('XF:11BMB-ES{Det:SAXS}:', name='pilatus300')
+    pilatus300.tiff.read_attrs = []
+    pilatus300.stats3.total.kind = 'hinted'
+    pilatus300.stats4.total.kind = 'hinted'
+    STATS_NAMES = ['stats1', 'stats2', 'stats3', 'stats4', 'stats5']
+    pilatus300.read_attrs = ['tiff'] + STATS_NAMES
+    for stats_name in STATS_NAMES:
+        stats_plugin = getattr(pilatus300, stats_name)
+        stats_plugin.read_attrs = ['total']
+
+    pilatus300.cam.ensure_nonblocking()
+else:
+    pilatus300 = 'Pil300ISNOTWORKING'
+
+#pilatus800 section
+if True:
```

Comment:
I think we should figure out a better way to do this. So that the profile collection doesn't need to be edited for each experiment.